autoapi/extension.py

```diff
@@ -135,10 +135,6 @@ def doctree_read(app, doctree):
             app.info(bold('[AutoAPI] ') +
                      darkgreen('Adding AutoAPI TOCTree [%s] to index.rst' % toc_entry)
                      )
-            if sphinx.version_info >= (1, 5):
```

Comment:
I don't believe we can just delete this..

Reply:
There was some explanation in #114 but I should have copied that over to here.
> Because we edit the toctree, I think the lines were added to ensure that the autoapi entry is correctly stored in the toctree, but the lines don't seem necessary. I've added a couple of tests to ensure that autoapi exists in the output toc.

I added the tests to test_integration.py.

Reply:
I feel okay with this change with the tests validating the toc 👍